### ISSUE TITLE
Trigger Re-Release

### ISSUE THREAD

**Issue by rctoris** [arebgun/dynamixel_motor#23]
Just a friendly request for a re-release into the build farm for Hydro and Indigo with the latest changes! :cake:

**Comment by 130s** [comment]
+1

@arebgun I'm constantly using your awesome dynamixel packages and would like it to be maintained with a bit more frequency. I can co-maintain to review pull requests as well as release tasks if you feel like.

**Comment by arebgun** [comment]
@130s, yes, I think if somebody else can co-maintain the package and do releases that would be great. How should we go about this? I guess I can add you as package committer here on github.

**Comment by 130s** [comment]
@arebgun Thanks, I see I've got a write access here now. Could you also add me a write access to https://github.com/arebgun/dynamixel_motor-release.git so that I can make a new release too?
